waspc/cli/src/Wasp/Cli/Interactive.hs

```diff
@@ -0,0 +1,80 @@
+module Wasp.Cli.Interactive
```

Comment:
Love the new naming! Interactive is pretty clear, and `askToChoose` and `askForInput` are great.